### DISCUSSION TITLE
How to rename snapshot folder?

### DISCUSSION THREAD

**Discussion by YuliyaMinsk** [vitest-dev/vitest#4211] [Q&A] [answered]
Hello!
I'm trying to create my first snapshot using Vitest. However, ls-lint doesn't seem to recognize folders with double underscores (__). How can I change the default folder for snapshots from `__snapshots__` to 'snapshots'?

Thanks in advance!

**Comment by sheremet-va** [comment, marked as answer]
https://vitest.dev/config/#resolvesnapshotpath
